aten/src/ATen/core/dynamic_type.cpp

```diff
@@ -374,7 +374,7 @@ DynamicTypePtr ivalue::TupleTypeFactory<c10::DynamicType>::fallback(
   return nullptr;
 }
 
-TORCH_API TupleTypePtr
+C10_UNUSED TORCH_API TupleTypePtr
 ivalue::TupleTypeFactory<TupleType>::fallback(const Type& type) {
```

Comment:
This is the intention, isn't it?
```suggestion
TORCH_API TupleTypePtr
ivalue::TupleTypeFactory<TupleType>::fallback(C10_UNUSED const Type& type) {
```